internal/common/disasters/commonpooldefinition.go

```diff
@@ -0,0 +1,55 @@
+package disasters
+
+import (
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+type Commonpool struct {
+	Resource, Threshold  uint
+}
+
+//islandContribution takes the resource donation from island to common pool
+func (e Environment) islandContribution(resource uint) {
+	e.CommonPool.Resource = resource
+}
+
+//disasterMitigate mitigates the disaster's damage using CP efore it hits the island
+func (e *Environment) DisasterMitigate(individualEffect map[shared.ClientID]float64, proportionalEffect map[shared.ClientID]float64) (map[shared.ClientID]float64, uint) {				//input is the porportional effect of each island
+	// compute total effect of disaster
+	totalEffect := 0.0
+	updatedIndividualEffect := map[shared.ClientID]float64{}
+
+	for _, effect := range individualEffect {
+		totalEffect = totalEffect + effect
+	}
+
+	// compute cp power towards disaster
+	var newmag float64
```

Comment:
what's `newmag`? New Magnitude? Let's call it `newMagnitude` :)

Reply:
yes new magnitude!

Reply:
I agree with having a more informative naming as well, following our camelCase conventions :)